sdks/python/src/opik/api_objects/OQL.py

```diff
@@ -0,0 +1,190 @@
+
+        else:
+            return {"field": field, "key": "", "type": COLUMNS[field]}
+
+    def _parse_operator(self, parsed_field: str) -> Dict[str, Any]:
+        # Skip whitespace
+        self._skip_whitespace()
+
+        # Parse the operator
+        if self.filter_string[self.i] == "=":
+            operator = "="
+            self.i += 1
+            return {"operator": operator}
+
+        elif self.filter_string[self.i] in ["<", ">"]:
+            if self.filter_string[self.i + 1] == "=":
+                operator = f"{self.filter_string[self.i]}="
+                self.i += 2
+            else:
+                operator = self.filter_string[self.i]
+                self.i += 1
+
+            if operator not in SUPPORTED_OPERATORS[parsed_field]:
+                raise ValueError(
+                    f"Operator {operator} is not supported for field {parsed_field}, only the operators {SUPPORTED_OPERATORS[parsed_field]} are supported."
+                )
+            return {"operator": operator}
+        else:
+            start = self.i
+            while (
+                self.i < len(self.filter_string)
+                and not self.filter_string[self.i].isspace()
+            ):
+                self.i += 1
+
+            operator = self.filter_string[start : self.i]
+            if operator not in ["contains", "not_contains"]:
+                raise ValueError(
+                    f"Operator {operator} is not supported for field {parsed_field}, only the operators {SUPPORTED_OPERATORS[parsed_field]} are supported."
+                )
+            return {"operator": operator}
+
+    def _get_number(self) -> str:
+        start = self.i
+        while self.i < len(self.filter_string) and self.filter_string[self.i].isdigit():
+            self.i += 1
+        return self.filter_string[start : self.i]
+
+    def _parse_value(self) -> Dict[str, Any]:
+        self._skip_whitespace()
+
+        start = self.i
+        if self.filter_string[self.i] == '"':
+            self.i += 1
+            start = self.i
+            while (
+                self.i < len(self.filter_string) and self.filter_string[self.i] != '"'
+            ):
+                self.i += 1
+
+            value = self.filter_string[start : self.i]
+
+            # Add 1 to skip the closing quote and return the value
+            self.i += 1
+            return {"value": value}
+        elif self.filter_string[self.i].isdigit() or self.filter_string[self.i] == "-":
+            value = self._get_number()
+            if self.i < len(self.filter_string) and self.filter_string[self.i] == ".":
+                self.i += 1
+                value += "." + self._get_number()
+
+            return {"value": value}
+        else:
+            raise ValueError(
+                f'Invalid value {self.filter_string[start:self.i]}, expected an string in double quotes("value") or a number'
+            )
+
+    def parse(self) -> Optional[str]:
```

Comment:
should be `_parse` since its called only in `OQL.__init__` method (and I don't think it will be needed to call it separately from the outside)